docs/source/examples/checks/performance/simple_model_comparison.ipynb

```diff
@@ -5,118 +5,84 @@
    "id": "840ea662",
```

Comment:
<p>second paragraph, change to:</p><p> </p><p>Note the we are demonstrating on a classification task, but the check also works for regression tasks. For classificaion we will see the metrics per class, and for regression we'll have a single result per metric</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>